app/models/customer.rb

```diff
@@ -0,0 +1,13 @@
+class Customer < ApplicationRecord
+    validates :first_name, presence: true
+    validates :last_name, presence: true
+    validates :phone, presence: true
```

Comment:
Since you are only validating the presence for these fields you can list them on one line.
`validates :first_name, :last_name, :phone, :presence: true`